scripts/lib/reload-shim.test.mjs

```
@@ -0,0 +1,196 @@
+    const result = runShim(shim, cleanEnvironment(root));
+    assert.equal(result.status, 0, result.stderr);
+    assert.equal(result.stdout.trim(), "tagged");
+  } finally {
+    if (server) {
+      server.close();
+      await once(server, "close");
+    }
+    try {
+      fs.unlinkSync(socketPath);
+    } catch (error) {
+      if (error.code !== "ENOENT") throw error;
```

Biome (2.5.6)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)